pkg/cmd/cli/config/smart_merge.go

```diff
@@ -140,3 +140,18 @@ func getMapKeys(theMap reflect.Value) (*util.StringSet, error) {
 	return ret, nil
 
 }
+
+func getUniqueName(basename string, existingNames *util.StringSet) string {
```

Comment:
To create the key names in the config files I think. @deads2k?

----- Original Message -----
> > @@ -140,3 +140,18 @@ func getMapKeys(theMap reflect.Value)
> > (*util.StringSet, error) {
> >  	return ret, nil
> >  
> >  }
> > +
> > +func getUniqueName(basename string, existingNames *util.StringSet) string
> > {
> 
> What does this get used for?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/992/files#r25139045